libkbfs/tlf_journal_test.go

```diff
@@ -494,7 +494,7 @@ func testTLFJournalBlockOpDiskByteLimit(t *testing.T, ver MetadataVer) {
 			ctx, id, bCtx, data2, serverHalf)
 	}()
 
-	numFlushed, rev, converted, err := tlfJournal.flushBlockEntries(ctx, 1)
+	numFlushed, rev, converted, err := tlfJournal.flushBlockEntries(ctx, 2)
```

Comment:
This was because i changed the start ordinal to 1, and flushBlockEntries errors if end <= first.